src/main/resources/cuda_header.h

```diff
@@ -1,28 +1,103 @@
 #define NVIDIA_WARP_SIZE 32 // this is typically 32 (for incl. 1080ti s)
```

Comment:
It would be better if the header file is protected to prevent unexpected multiple includes.

Reply:
Good point.